pom.xml

```diff
@@ -362,15 +362,6 @@ THE SOFTWARE.
           <antlr4>
             <antlr4Formatter />
           </antlr4>
-          <groovy>
-            <includes>
-              <include>Jenkinsfile</include>
-              <include>licenseCompleter.groovy</include>
-            </includes>
-            <greclipse>
-              <file>config/greclipse.properties</file>
-            </greclipse>
```

Comment:
Is the dependency on `shared-resources` a few lines below still needed now that `config/greclipse.properties` is no longer referenced by the configuration?

Reply:
Indeed, https://github.com/jenkinsci/jenkins/pull/8106